lib/iruby/kernel.rb

```diff
@@ -109,7 +109,7 @@ def error_message(e)
     def complete_request(msg)
       # HACK for #26, only complete last line
       code = msg[:content]['code']
-      if start = code.rindex("\n")
+      if start = code.rindex(/\s|\R/)
```

Comment:
`\R` means linebreak.

https://github.com/k-takata/Onigmo/blob/master/doc/RE#L101-L107
(onigumo is the current regex engine in Ruby 2.x)